### DISCUSSION TITLE
Don't skip versions when suggesting updates

### DISCUSSION THREAD

**Discussion by OuesFa** [renovatebot/renovate#15480] [Request Help]
Is it possible to tell renovate not to skip a version when it proposes updates for a given package? 

Example, I have a dockerfile package with `FROM mariadb:10.4.22`
and renovate proposes me directly the `10.8.x`.
How to make it propose me the `10.5.x` at the most? 

Thanks

**Comment by rarkins** [comment]
Not yet. See #6576

**Reply by OuesFa** [reply]
Thanks for your answer, actually my current problem is that I'm trying to activate renovate on a repo that already exists and for which the existing packages are far behind the upstream versions, I'd like to avoid having to do big updates all at once, and do one version at a time. 
Once my repo has caught up with the upstream package versions, I don't mind if Renovate only suggests the latest versions, but in the meantime I'd like to avoid this "big update effect". 
 
I thought for example to duplicate the folder that contains my packages, leaving one "managed by renovate" that will propose the latest versions that I merge to have my docker images in my registry. 
And continue on the "manual" folder to update my images until the day I catch up with the oldest version updated by Renovate. 

Do you have a (better) strategy in mind to handle this?

Thanks!

**Reply by viceice** [reply]
1. enable dashboard approval for major & minor updates and separate minor patch. then you can see all updates and get patch updates right away
2. approve updates which are ok
3. add package rules for updates with allowed version for packages you want lower versions for
4. goto 2.

**Reply by OuesFa** [reply]
Thanks for your thoughts, I'll try it out.
Dashboard approval won't be possible though because issues are disabled on the repo 😬
